### PR TITLE
chore: executable curl of SQL http

### PR DESCRIPTION
**Pull request by nicecui**

### CODE REVIEW

**Review by sunng87 — APPROVED** [review]
LGTM

File: docs/reference/sql/http-api.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+http://{{API-host}}/v1/sql?db={{db-name}}
+```
+
+#### Method
+
+Use the POST method to submit the SQL statement to the server.
+
+#### Header
+
+* `Authorization`: The credential. Required if you are using GreptimeDB with authorized users. Please refer to [authentication](/user-guide/clients.md#HTTP-API).
```
Comment (sunng87) [inline docs/reference/sql/http-api.md:19]: > using GreptimeDB with authorized users

can be misleading with *started greptime with particular operation system user*. How about *with authentication enabled`

File: docs/user-guide/write-data.md [hunk 2]
```diff
@@ -220,3 +219,40 @@ curl -X POST http://127.0.0.1:4000/v1/opentsdb/api/put -d '
 
 See [Prometheus Storage](./prometheus.md#storage) to know how to write data.
 
+## Delete
+
+### SQL
+
+#### `Delete` Statement
```
Comment (sunng87) [inline docs/user-guide/write-data.md:226]:
```suggestion
#### `DELETE` Statement
```

for consistency

File: docs/reference/sql/http-api.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# HTTP API
+
+To submit a SQL statement to the GreptimeDB server via HTTP API, use the following format:
```
Comment (sunng87) [inline docs/reference/sql/http-api.md:3]:
```suggestion
To submit a SQL query to the GreptimeDB server via HTTP API, use the following format:
```

File: docs/user-guide/query-data.md [hunk 4]
```diff
@@ -6,28 +6,29 @@ GreptimeDB supports full SQL for you to query data from a database. Here are som
 To select all the data from the `system_metrics` table, use the `SELECT` statement:
```
Comment (xtang) [inline docs/user-guide/query-data.md:6]:
```suggestion
To select all the data from the `monitor` table, use the `SELECT` statement:
```

And the sentence above need to be modified too

**Comment by xtang** [comment]
almost lgtm. Thanks @nicecui

File: docs/user-guide/query-data.md (hunk 4 above)
Comment (nicecui) [reply docs/user-guide/query-data.md]: sharp eyes 👍

**Review by xtang — APPROVED** [review]
LGTM

**Pull request merged**